doc/manual/config-basic.rst

```diff
@@ -107,6 +107,149 @@ Problems can have special *compare* and
 *run* scripts associated to them, to deal with problem
 statements that require non-standard evaluation.
 
+Adding contest data programmatically
+------------------------------------
```

Comment:
Is config-basic a good place or is a separate “Import/Export” chapter warranted?

Reply:
I have moved it into its own chapter.